### ISSUE TITLE
exctracteWire5.m hard coded parameter

### ISSUE THREAD

**Issue by gBringout** [gBringout/CoilDesign#2]
The hard coded parameter 'length1(index)' on line https://github.com/gBringout/CoilDesign/blob/master/streamBEM/otherFunctions/exctracteWire5.m#L138 should be made variable.

**Comment by hbbdyzf** [comment]
the extracteWire5 is not stable. it is not suitable the wire or stream has big wave.

**Comment by gBringout** [comment]
Correct, as indicated in the [Thesis ](https://zenodo.org/record/162417/) on chapter 4.1.4.
